packages/pigeon/lib/src/dart/templates.dart

```diff
@@ -87,6 +87,9 @@ class $dartInstanceManagerClassName {
   late final void Function(int) onWeakReferenceRemoved;
 
   static $dartInstanceManagerClassName _initInstance() {
+    if (Platform.environment['FLUTTER_TEST'] == 'true') {
```

Comment:
Hm, is `FLUTTER_TEST` only set in unit tests? Or will this break `flutter test` integration test behavior?

Reply:
I forgot that integration tests also use `flutter test`. But I just tried it and it looks like this doesn't affect integration tests. It seems the environment variable is only for unit tests.